metricbeat/module/windows/perfmon/reader.go

```diff
@@ -124,6 +129,44 @@ func (re *Reader) Read() ([]mb.Event, error) {
 	return events, nil
 }
 
+func (re *Reader) getValues() (map[string][]pdh.CounterValue, error) {
+	var interval uint32 = 10
+	var val map[string][]pdh.CounterValue
+	rand.Seed(time.Now().UnixNano())
+	title := windows.StringToUTF16Ptr(randSeq(10))
```

Comment:
why do we need a random title here? must this be unique?

Reply:
Can we prefix it so at least an administrator can identify the source?

Reply:
added a prefix to it